tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -137,8 +142,8 @@ spec:
         - validate
         - image
         - "--verbose"
-        - "--json-input"
-        - "$(params.IMAGES)"
+        - "--file-input"
```

Comment:
An alternative is to introduce a new flag, e.g. `--images`, that can take a file path or an inline snapshot, kind of like the `--policy` flag. That will give us a lot of flexibility, and allow the task to remain backwards compatible.